Bottom: 8f011c83c9391edc8b1e0a751077fe4869522955
Top:    4bbad2bddea8617dca5ef5e574fb13b38ab289ef
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-04 09:50:49 -0500

Refresh of initial

---

diff --git a/farm-compile b/farm-compile
index 8cb958f..32196dd 100755
--- a/farm-compile
+++ b/farm-compile
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /bin/sh
 
 set -e
 source ~/.farmrc
diff --git a/farm-config32 b/farm-config32
index 07f71e5..55c390a 100755
--- a/farm-config32
+++ b/farm-config32
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /bin/sh
 
 set -e
 source ~/.farmrc
diff --git a/farm-config64 b/farm-config64
index 350317a..7e8b82d 100755
--- a/farm-config64
+++ b/farm-config64
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /bin/sh
 
 set -e
 source ~/.farmrc
diff --git a/farm-hardlink b/farm-hardlink
index 8c340da..11054d1 100755
--- a/farm-hardlink
+++ b/farm-hardlink
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /bin/sh
 
 set -e
 source ~/.farmrc
diff --git a/farm-warnings b/farm-warnings
index 8cbe679..83d959c 100755
--- a/farm-warnings
+++ b/farm-warnings
@@ -1,3 +1,3 @@
-#! /bin/bash
+#! /bin/sh
 
 grep -E -h -i '(error|warning):' Log* | grep -v /Module.symvers |sort -u
